connector/src/main/scala/quasar/qscript/Transform.scala

```diff
@@ -649,14 +567,39 @@ class Transform
           JoinType.LeftOuter,
           LeftSideF)).embed).right
 
-    case lp.InvokeUnapply(func @ TernaryFunc(Transformation, _, _, _, _, _, _), Sized(a1, a2, a3)) =>
-      invokeThetaJoin(
-        Func.Input3(a1, a2, a3),
-        func match {
-          case set.InnerJoin      => JoinType.Inner
-          case set.LeftOuterJoin  => JoinType.LeftOuter
-          case set.RightOuterJoin => JoinType.RightOuter
-          case set.FullOuterJoin  => JoinType.FullOuter
-        })
+    case lp.JoinSideName(name) =>
+      Target(
+        Ann(Nil, StrLit(name.toString)),
```

Comment:
We turn the `JoinSideName` into a `StrLit`. This is broken because the string `name.toString` might already exist in the query.